en/flying/led_meanings.md

```diff
@@ -0,0 +1,38 @@
+# LED meanings
+
+[Pixhawk flight controllers](../flight_controller/pixhawk_series.md) have an RGB LED to indicate the current status of the FMU.
+
+![LED meanings](../../images/led_meanings.gif)
```

Comment:
Possible changes to diagram:
- Put the green first and flashing green below that. Ie order from most ready to least ready to fly.
- Next to the green - "Armed, GPS (Ready to fly)"

Reply:
I got this diagram and the description from http://readme.dronesmith.io/docs/fmu-status-led with Jinger's permission.
If it is important, I'll look into a suitable gif editor to crop it.